### DISCUSSION TITLE
Web3Auth Flutter SDK + Self host authentication

### DISCUSSION THREAD

**Discussion by iNima** [Web3Auth/Web3Auth#763] [Help]
Hi there,
Is it possible to use self host authentication approach with the Web3Auth Flutter SDK?
All I need is to be able to do the reconstruction of the keys in the background without redirecting to a web page.

I read in the discussion that the solution would be using the self host authentication instead of the plug n play SDK.
[https://github.com/Web3Auth/web3auth-web/discussions/449#discussioncomment-3792829](url)

But I don't know how to do that for a Flutter app. It seems the self hosted auth documentation is just related to the Web3Auth-Web sdk, right?

**Comment by himanshuchawla009** [comment]
Hey currently it is not possible to self host in mobile, we are working on self hosting sdks for mobile stack.

**Reply by iNima** [reply]
Understood! Is there any ETA for this feature on Flutter SDK?

**Reply by masehdh** [reply]
Any update on this for flutter? @himanshuchawla009
